e2e/tests/features/sdk_website.feature

```diff
@@ -15,16 +13,19 @@ Feature: SDK Demo Site
   Background:
     Given Project is protect-and-track-demo
 
-  @regression
-  Scenario Outline: Decrypt TDF3 file in SR
-    Given a TDF3 file: <filename>
+  @smoke
+  Scenario Outline: Encrypt file in Protect and Track demo site
+    Given an attachment file: <filename>
     When I start to run testrail <case_id>
-    And qavirtru31@cmk-test-org.com logged in secure-reader page using a new browser
-    And the user drag and drop the file in secure-reader page
-    And the file automatically decrypt in secure-reader page
-    Then screen comparison shall have similarity(0.0-1.0) greater than 0.999
+    And I opened protect-and-track page using a new browser
+    And I drag and drop the file in protect-and-track page
```

Comment:
@benlstanley according to the site `the user drag and drop the file in (secure-reader, gmail, protect-and-track) page`

Reply:
Ok good catch we will update the confluence page!